src/pages/inventory/AllCategories.tsx

```diff
@@ -0,0 +1,83 @@
+// import React from 'react';
```

Comment:
this one is waiting for create product is it

Reply:
the 3 categories files i did then i realised no need... but i want to leave it in case i need the components for the actual CRUD categories via product page